apps/src/blocklyAddons/cdoBlockSvg.js

```diff
@@ -3,7 +3,7 @@ import BlockSvgUnused from './blockSvgUnused';
 
 export default class BlockSvg extends GoogleBlockly.BlockSvg {
   constructor(workspace, prototypeName, opt_id) {
-    super(workspace, prototypeName, opt_id);
+    super(workspace, prototypeName, ++Blockly.uidCounter_); // Use counter instead of randomly generated IDs
```

Comment:
Unlikely, but is there any risk of overflow?

Reply:
hmm, I guess it's possible? It's the same setup we've used for years with CDO Blockly (https://github.com/code-dot-org/blockly/blob/main/core/ui/block.js#L47) so I'm sort of inclined to just stick with it this way